### PR TITLE
Added env variable JVM_MEM_OPTS

### PR DESCRIPTION
**Pull request by BLasan**

### CODE REVIEW

File: artifacts/operator-configs/controller-conf.yaml [hunk 1]
```diff
@@ -58,6 +59,8 @@ data:
   p1-apim-deployment-resources-requests-memory: "2Gi"
   p1-apim-deployment-resources-limits-cpu: "2000m"
   p1-apim-deployment-resources-limits-memory: "2Gi"
+  p1-apim-deployment-env-jvm-heap-memory-xms: "1024m"
+  p1-apim-deployment-env-jvm-heap-memory-xmx: "1024m"
```
Comment (pubudu538) [inline artifacts/operator-configs/controller-conf.yaml]: Lets add this as a global value for all the patterns. Simply we can drop p1 from this variable.

**Review by pubudu538 — CHANGES_REQUESTED** [review]

**Comment by BLasan** [comment]
@pubudu538 Changes added. Please review

**Review by pubudu538 — APPROVED** [review]

File: artifacts/operator-configs/controller-conf.yaml [hunk 2]
```diff
@@ -74,6 +78,7 @@ data:
   p1-apim-analytics-deployment-resources-limits-cpu: "2000m"
   p1-apim-analytics-deployment-resources-limits-memory: "4Gi"
 
+
```
Comment (Arshardh) [inline artifacts/operator-configs/controller-conf.yaml:81]: Let's try to avoid adding additional lines than necessary.

Comment (BLasan) [reply artifacts/operator-configs/controller-conf.yaml:81]: fixed via https://github.com/wso2/k8s-wso2am-operator/pull/90/commits/dd336c4637bd040f1753f012f46578f0cc703ac3

**Pull request merged**